src/DssSpell.t.sol

```diff
@@ -975,6 +1042,138 @@ contract DssSpellTest is DssSpellTestBase {
         }
     }
 
+    // Note: this test deliberately ignore changes in SKY and MKR held in the treasury and their supply.
+    // TODO: Consider simplifying `testPayments` above to be closer to this one.
+    // There is coverage for them in the spell-specific tests.
+    function testPayments_ignoreBurnsAndConversions() public {
+        Payee[12] memory payees = [
+            Payee(address(usds), addr.addr("SPARK_PROXY"),               20_600_000 ether), // Note: ether is only a keyword helper
```

Comment:
Maybe a potential improvement for future spells. Instead of using `ether` and having to append the `//Note` everywhere a cleaner approach would be to define functions for the most used tokens, then the usage would be:
```
Payee(address(sky),  wallets.addr("LAUNCH_PROJECT_FUNDING"), sky(8_400_000)),
Payee(address(usds), wallets.addr("BLUE"),                   usds(54_167)),
```
Alternative, this can also be defined as a lib, so we get a more "english" like syntax:
```
Payee(address(sky),  wallets.addr("LAUNCH_PROJECT_FUNDING"), 8_400_000.sky()),
Payee(address(usds), wallets.addr("BLUE"),                   54_167.usds()),
```
But I'm not sure if it's worth the effort.